### ISSUE TITLE
Getting Json files instead of pictures

### ISSUE THREAD

**Issue by reefor** [pinokiofactory/aura-sr-upscaler#1]
This is a strange problem. Instead of getting picture files I'm getting Json files.

I uninstalled & reinstalled several times, I used my own jpgs & pngs, and I used the sample pictures provided & got the same result each time.

I don't know if json files can be converted into picture files, but if they can I am sure I don't have the software to do it. lol

Is this a known bug, or am I somehow doing something wrong?

**Comment by DreadFox** [comment]
Yeah, same here. the comparison tool/slider basically shows the placeholder for 'image failed to load', and the download icon in that area returns a  .json file.

![image](https://github.com/user-attachments/assets/b0e61eb7-4df0-4f25-ad0e-1e26ecf0777d)

clicking the download button and attempting to save the .json results in no file being saved on the PC.  The link it is downloading from suggests that it should be 'image.webp' rather than 'image.json'

![image](https://github.com/user-attachments/assets/2aa59802-9d90-4d4c-bf85-8d764154423f)

note that the demo on Hugging Face works fine, might just be a bad path or something? 
https://huggingface.co/spaces/gokaygokay/AuraSR-v2

**Comment by WolfLycanorcant** [comment]
same here...  its not giving me the interface in pinokio and says i need another app to open the interface. When i connect to the network address, it gives me a json file to download.
